templates/volunteer/volunteerprofile_form.html

```diff
@@ -5,38 +5,56 @@
-            {% if object.pk %}
-                Update your information below.
-            {% else %}
-                Fill in the profile below.
-            {% endif %}
-            Fill in the profile below.
+            {% if object.pk %}Update your information below.{% else %}Fill in the profile below.{% endif %}
         </p>
         <div class="row g-5">
             {% bootstrap_form_errors form %}
-            <form action="{% if object.pk %}{% url 'volunteer:volunteer_profile_edit' object.pk %}{% else %}{% url 'volunteer:volunteer_profile_new' %}{% endif %}"
-                  method="post"
-                  class="form">
-                {% csrf_token %}
-                {% bootstrap_field form.languages_spoken %}
-                {% bootstrap_field form.teams %}
-                {% bootstrap_field form.pyladies_chapter %}
-                {% bootstrap_field form.timezone %}
-                {% bootstrap_field form.github_username %}
-                {% bootstrap_field form.discord_username %}
-                {% bootstrap_field form.instagram_username %}
-                {% bootstrap_field form.bluesky_username %}
-                {% bootstrap_field form.mastodon_url %}
-                {% bootstrap_field form.x_username %}
-                {% bootstrap_field form.linkedin_url %}
-                <input type="submit"
-                       value="{% if object.pk %}Update Profile{% else %}Create Profile{% endif %}"
-                       class="btn btn-primary" />
+            <form action="{% if object.pk %}{% url 'volunteer:volunteer_profile_edit' object.pk %}{% else %}{% url 'volunteer:volunteer_profile_new' %}{% endif %}" method="post" class="form">{% csrf_token %}
+                <div class="card mb-4">
+                    <div class="card-header">
+                        <h5 class="card-title mb-0">Basic Information</h5>
+                    </div>
+                    <div class="card-body">
+                        {% bootstrap_field form.languages_spoken %}
+                        {% bootstrap_field form.teams %}
+                        {% bootstrap_field form.pyladies_chapter %}
+                        {% bootstrap_field form.timezone %}
+                    </div>
+                </div>
+                
+                <div class="card mb-4">
+                    <div class="card-header">
+                        <h5 class="card-title mb-0">Social Media Profiles</h5>
+                        <p class="text-muted small mt-2 mb-0">Please enter your usernames without the '@' symbol unless specified otherwise.</p>
+                    </div>
+                    <div class="card-body">
+                        <div class="row">
+                            <div class="col-md-6">
+                                {% bootstrap_field form.github_username %}
+                                {% bootstrap_field form.discord_username %}
+                                {% bootstrap_field form.instagram_username %}
```

Comment:
Perhaps we could render some of these with bootstrap addons, so that it is clear that the usernames should not include the `@` anymore. (And still adding the validation logic that you've created)

https://getbootstrap.com/docs/5.3/forms/input-group/#basic-example

https://django-bootstrap-v5.readthedocs.io/en/latest/templatetags.html